.scripts/deploy_gcloud.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -o pipefail
+set -o errexit
+set -o nounset
+# set -o xtrace
+
+# Set magic variables for current FILE & DIR
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(cd "$(dirname "${__dir}")" && pwd)"
+
+# Store the new image in docker hub
+docker build --quiet -t kiloreux/tusd:latest -t kiloreux/tusd:$TRAVIS_COMMIT ${__root};
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+docker push kiloreux/tusd:$TRAVIS_COMMIT;
+docker push kiloreux/tusd:latest;
```

Comment:
Could this be tusproject/tusd?

Reply:
It could be of course. But I don't have the permissions there and It was already late so no one will grant me that. I figured I would go with my own setup and experiment on building and then we will just push images to the tusproject/tusd repository

Reply:
iirc, we use docker automated builds to push to tusproject/tusd or no? so that will collide then. I do agree of course that we should have just one container, and that is `tusproject/tusd`, but we need to figure out how to best piece this all together then

Reply:
> we use docker automated builds to push to tusproject/tusd or no?

I am not sure whether this feature got finished, but I actually prefer using Travis CI for pushing the new docker container. This keeps the test/build/deploy process on one service instead of using Travis and Docker.